### ISSUE TITLE
What if victory status integrated to overview?

### ISSUE THREAD

**Issue by saud2410** [yairm210/Unciv#3417]
I always thought what if i could see victory status with [ overview ] button.

I look overview and victory status a lot - nearly every turn after turn 2~30 for diplomatics. It's very profitable introducing two civs at the early game, or requesting one civ to declare war to another civ. To do so I need to look diplomatic window, overview and ranking a lot. (I know i too prefer diplomatic than war, and i'm still noob, but it can't help for now)

Some may not look victory status and overview a lot. And there must be reason they're separated. But they looks so similar to me. If they are not integrated, i want a button or something to open victory status(especially rankings) for one touch.

**Comment by yairm210** [comment]
I can't think of a good way visually to incorporate this. It can't fit in the overview, and I'm not sure if 'linking out' would be comfortable. 
What were you thinking of?

**Comment by krushia** [comment]
To me the awkwardness is having to open the menu to get to victory status, whereas every other screen related to the current game is accessible from a button at the top of the map screen.

**Comment by yairm210** [comment]
Again, I understand the problem, what do you suggest?

**Comment by Mugi1234** [comment]
Just merge it with the overview it will be awesome

**Comment by saud2410** [comment]
I think they are organically related and there are plenty of space, so imao merging is better. But if it's hard linking out is also grate. There are already victory condition hotkey on keyboard, and mobile needs it too.

What about nation name button links to victory condition? There are already many ways to go overview directly.

![Screenshot_20210622-010603_UnCiv](https://user-images.githubusercontent.com/64586749/122795723-9183fa00-d2f8-11eb-8928-6884989e24ec.jpg)

**Comment by AdityaMH** [comment]
#5856

**Comment by Thienanbad** [comment]
this is from 2020. Now the Victory status tab still not access easier

**Comment by PhilipPurwoko** [comment]
I would agree to integrated into overview menu or add an icon on the left of the overview button to indicate the victory progress

**Comment by yairm210** [comment]
The second

**Comment by yairm210** [comment]
But we still need to allow endgame screen to be its own thing
